x-pack/metricbeat/module/aws/cloudwatch/cloudwatch_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/aws/aws-sdk-go-v2/service/apigateway"
+	restapi "github.com/aws/aws-sdk-go-v2/service/apigateway"
```

Comment:
We can remove `restapi "github.com/aws/aws-sdk-go-v2/service/apigateway"` and just use "github.com/aws/aws-sdk-go-v2/service/apigateway"

Reply:
Was a leftover, indeed.
Removed